pkg/transformer/kubernetes/k8sutils_test.go

```diff
@@ -296,3 +297,29 @@ func TestSortedKeys(t *testing.T) {
 		t.Logf("Test Fail output should be %s", c)
 	}
 }
+
+//test conversion from duration string to seconds *int64
+func TestDurationStrToSecondsInt(t *testing.T) {
+	testCases := map[string]struct {
+		in  string
+		out *int64
+	}{
+		"5s":      {in: "5s", out: &[]int64{5}[0]},
```

Comment:
Sure. Stop_grace_period is stirng type so quotes are needed.
And from[Specifying durations](https://docs.docker.com/compose/compose-file/compose-file-v2/#orig-resources)
```
stop_grace_period: '2'
```
should be invalid and has no effect.